src/components/slop-fest-modal.jsx

```diff
@@ -0,0 +1,210 @@
+              labelText="Name it!"
+              className={`w-[373px]`}
+              id={"name"}
+              register={register("name", {
+                required: true,
+                pattern: {
+                  value: /^\S/,
+                  message: "Must not start with a space",
+                },
+              })}
+              onChange={(evt) => {
+                setValue("name", evt.target.value, { shouldValidate: true });
+              }}
+              isValid={isValid}
+            />
+            {errors.name ? (
+              <Form.Feedback
+                message={"Must contain more than one character"}
+                className={""}
+              />
+            ) : (
+              ""
+            )}
+          </div>
+          <div className="flex flex-row justify-between">
+            <div className="flex-col">
+              <Form.DateDropdown
+                labelText={"Start Date"}
+                className={"w-[176px]"}
+                required
+                name="startDate"
+                isValid={isValid}
+                date={startDate}
+                onChange={(date) => {
+                  setStartDate(date);
+                  setValue("startDate", date, {
+                    shouldValidate: true,
+                  });
+                }}
+              />
+              {errors.startDate ? (
+                <Form.Feedback message={"Must be a valid date"} />
+              ) : (
+                ""
+              )}
+            </div>
+            <div className="flex-col">
+              <Form.DateDropdown
+                labelText={"End Date"}
+                className={"w-[176px] h-12"}
+                required
+                name="endDate"
+                isValid={isValid}
+                date={endDate}
+                onChange={(date) => {
+                  setEndDate(date);
+                  setValue("endDate", date, {
+                    shouldValidate: true,
+                  });
+                }}
+              />
+              {errors.endDate ? (
+                <Form.Feedback message={"Must be a valid date"} />
+              ) : (
+                ""
+              )}
+            </div>
+          </div>
+          <div className="flex font-bold font-arial flex-col py-3 border-solid rounded-none border-black/[0.4]">
+            <label htmlFor="attending" className="mb-1.5 text-lg">
+              Goblins Attending
+            </label>
+            {/* will change to headless ui component */}
+            <Select
+              options={userOptions}
+              className={"basic-multi-select"}
+              classNames={{
+                control: () =>
+                  "w-[373px] max-h-[92px] px-2 py-2 border border-solid rounded-none border-black overflow-scroll",
+              }}
+              isMulti
+              name="attendees"
+              isSearchable={true}
+              isClearable={false}
+              // onChange={(evt) => {
```

Comment:
Attendees State Management: The form doesn't appear to properly handle changes to the attendees from the Select component. You should ensure that the onChange handler updates the form's state appropriately.

For example, only one user can be added to a slop at a time.

onChange={(selectedOptions) => {
  const attendees = selectedOptions.map(option => option.value);
  setValue('attendees', attendees, { shouldValidate: true });
}}

Reply:
Yeah I didn't finish setting that up because I was waiting for the form combo box to be completed. We decided that the form combo box would replace that component currently in the  slop fest modal.

Reply:
Ahhh gotcha!